src/dialogs_win.cc

```diff
@@ -12,68 +12,66 @@ HRESULT __hr__;
 }                                                
 #endif  // CHECK
 
-
-// { WINSTRING
-WinString::WinString(const std::string &string) {
-  std::wstringstream ss;
-    ss << s2ws(string);
-    ss >> str;
-}
-
-std::string WinString::operator()() {
-  std::string res;
-  if (str != nullptr) {
-    std::wstring ss(str);
-    res = ws2s(ss);
-  }
-  return res;
-}
-
-// http://stackoverflow.com/questions/4804298/how-to-convert-wstring-into-string
-std::wstring WinString::s2ws(const std::string& str) {
-  typedef std::codecvt_utf8<wchar_t> convert_typeX;
-  std::wstring_convert<convert_typeX, wchar_t> converterX;
-  return converterX.from_bytes(str);
-}
-
-std::string WinString::ws2s(const std::wstring& wstr) {
-  typedef std::codecvt_utf8<wchar_t> convert_typeX;
-  std::wstring_convert<convert_typeX, wchar_t> converterX;
-  return converterX.to_bytes(wstr);
-}
-
-// WINSTRING }
-
 // { COMMON_DIALOG
 CommonDialog::CommonDialog(CLSID type) : dialog(nullptr) {
     check(CoCreateInstance(type, nullptr, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&dialog)), "Failed to create instance");
     check(dialog->GetOptions(&options), "Failed to get options from instance");
 }
-
-void CommonDialog::set_title(const std::string &title) {
-    auto tmp = std::wstring(title.begin(), title.end());
-    auto t = tmp.data();
-    check(dialog->SetTitle(t), "Failed to set dialog title");
+void CommonDialog::set_title(std::string &&title) {
```

Comment:
In general, that is, 99.9% of the cases or so, we do not need to implement or use move operator specifically unless we for instance write our own container that we use. I'll fix this in #80.